hnn_core/gui/gui.py

```diff
@@ -1223,7 +1375,68 @@ def add_connectivity_tab(params, connectivity_out,
     with connectivity_out:
         display(cell_connectivity)
 
-    return net
+    return network
+
+
+def add_cell_parameters_tab(network, cell_params_out, cell_pameters_vboxes,
+                            cell_layer_radio_button, cell_type_radio_button,
+                            layout):
+    L2_defautl_values = get_L2Pyr_params_default()
+    L5_default_values = get_L5Pyr_params_default()
+    cell_types = [("L2", L2_defautl_values), ("L5", L5_default_values)]
+    style = {'description_width': '235px'}
+    kwargs = dict(layout=layout, style=style)
+
+    # get_cell_params_dic_values(network.cell_types)
+    for cell_type in cell_types:
+        layer_parameters = list()
+        for layer in cell_parameters_dict.keys():
+            if 'Biophysic' in layer and cell_type[0] not in layer:
+                continue
+
+            for parameter in cell_parameters_dict[layer]:
+                param_name, param_units, params_key = (parameter[0],
+                                                       parameter[1],
+                                                       parameter[2])
+                default_value = get_cell_param_default_value(
+                    f'{cell_type[0]}Pyr_{params_key}', cell_type[1])
+                description = f"{param_name} ({param_units})"
+                min_value = -1000.0 if param_units not in 'ms' else 0
+                text_field = BoundedFloatText(value=default_value,
+                                              min=min_value,
+                                              max=1000.0,
+                                              step=0.1,
+                                              description=description,
+                                              disabled=False,
+                                              **kwargs)
+                layer_parameters.append(text_field)
+            cell_pameters_key = f'{cell_type[0]}_{layer}'
+            cell_pameters_vboxes[cell_pameters_key] = VBox(layer_parameters)
+            layer_parameters.clear()
+
+    # clear existing connectivity
+    cell_params_out.clear_output()
+
+    # Add cell parameters
+    display(_update_cell_params_vbox(cell_params_out,
+                                     cell_pameters_vboxes,
+                                     cell_type_radio_button.value,
+                                     cell_layer_radio_button.value))
+
+
+def get_cell_param_default_value(cell_type_key, param_dict):
+    return param_dict[cell_type_key]
+
+
+def get_cell_params_dic_values(network_cell_types):
```

Comment:
seems like this function isn't being used anywhere

Reply:
I removed it